src/payment/braintree.md

```diff
@@ -185,7 +185,7 @@ The following descriptors are used to identify purchases on customer credit card
 
 {:.bs-callout-warning}
 **<a name="migration">Magento 2.4 Migration Notes:</a>**<br />
-Before the release of Magento 2.4.0, we recommended that merchants replace the core Magento Braintree integration with the official Braintree extension from [Magento Marketplace](https://marketplace.magento.com/catalogsearch/result/?q=braintree). Beginning with Magento 2.4.0, the Braintree extension is included in the release.<br/><br/>
+Before the release of Magento 2.4.0, we recommended that merchants replace the core Magento Braintree integration with the official Braintree extension from [Magento Marketplace](https://marketplace.magento.com/catalogsearch/result/?q=braintree). Beginning with Magento 2.4.0, the Braintree extension is not included in the release.<br/><br/>
```

Comment:
@ihor-sviziev  The original native integration was removed. However, the marketplace extension is now part of the install as a "bundled" extension. Do you mean that it is not "enabled" by default?